docs/reference/configuration-parameters.rst

```diff
@@ -0,0 +1,10 @@
+========================
+Configuration parameters
+========================
+
+This section contains a list of properties that can be configured in ``scylla.yaml`` - the main configuration file for ScyllaDB.
```

Comment:
would be better if we can have a link to `operating-scylla/system-configuration/` ?